### PR TITLE
JAMES-2096 Simple migration task

### PR DESCRIPTION
**Pull request by chibenwa**
Let see later if a more complicated logic is needed...

(IE if we need to // it)

Review only last commit please

### CODE REVIEW

File: mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageDAOV2.java [hunk 1]
```diff
@@ -208,6 +209,55 @@ private UDTValue toUDT(MessageAttachment messageAttachment) {
...
+            .setString(BODY_CONTENT, pair.getRight().map(BlobId::getId).orElse(DEFAULT_OBJECT_VALUE))
+            .setString(HEADER_CONTENT, pair.getLeft().map(BlobId::getId).orElse(DEFAULT_OBJECT_VALUE))
+            .setLong(TEXTUAL_LINE_COUNT, message.getTextuaLineCount())
+            .setList(PROPERTIES, buildPropertiesUdt(message.getPropertyBuilder().toProperties()))
+            .setList(ATTACHMENTS, buildAttachmentUdt(message));
+    }
+
+    private List<UDTValue> buildPropertiesUdt(MailboxMessage message) {
+        return message.getProperties().stream()
+            .map(x -> typesProvider.getDefinedUserType(PROPERTIES)
```
Comment (quynhn) [inline mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageDAOV2.java]: can we more meaning for "x"?

File: mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/migration/V1ToV2MigrationTest.java [hunk 2]
```diff
@@ -166,6 +166,44 @@ public void migrationShouldWorkWithoutAttachments() throws Exception {
...
+
+        CassandraMessageDAOV2.MessageResult messageResult = retrieveMessageOnV2().get();
+        softly.assertThat(messageResult.message().getLeft().getMessageId()).isEqualTo(messageId);
+        softly.assertThat(IOUtils.toString(messageResult.message().getLeft().getContent(), Charsets.UTF_8))
+            .isEqualTo(CONTENT);
+        softly.assertThat(messageResult.message().getRight().findAny().isPresent()).isFalse();
+    }
+
+    @Test
+    public void ruwShouldBeIndempotent() throws Exception {
```
Comment (quynhn) [inline mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/migration/V1ToV2MigrationTest.java]: s/Indempotent/Idempotent/g?

**Comment by chibenwa** [comment]
See https://github.com/linagora/james-project/pull/906

File: mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/migration/V1ToV2Migration.java [hunk 3]
```diff
@@ -120,5 +120,7 @@ private void migrate(CassandraMessageDAO.RawMessage rawMessage) {
         messageDAOV2.save(rawMessage)
             .thenCompose(any -> messageDAOV1.delete(messageId))
             .join();
+
+        LOGGER.info("{} migrated", rawMessage.getMessageId());
```
Comment (rouazana) [inline mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/migration/V1ToV2Migration.java]: debug?

File: mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/migration/V1ToV2MigrationTest.java [hunk 4]
```diff
@@ -166,6 +166,44 @@ public void migrationShouldWorkWithoutAttachments() throws Exception {
     }
 
     @Test
+    public void ruwShouldMigrateMessages() throws Exception {
```
Comment (rouazana) [inline mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/migration/V1ToV2MigrationTest.java]: s/ruw/run/ (and later)

**Pull request closed without merging**